functions/go/gatekeeper-validate/README.md

```diff
@@ -0,0 +1,41 @@
+# gatekeeper-validate
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Validate the KRM resources using the policy controller.
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+You can use the policy controller to validate KRM resources. To learn more about
+the policy controller, see: https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller.
+
+The function ensures the constraint policies are enforced on KRM resources.
```

Comment:
nit: "The function ensures the constraint policies are enforced on KRM resources." -> "The function evaluates constraint policies against KRM resources"